about.html

```diff
@@ -0,0 +1,81 @@
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+    <title>About - CEC </title>
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#">Navbar</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarTogglerDemo02">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="/index.html">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/about.html">About me</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/contact.html">Contact me</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/projects.html">projects</a>
+                        </li>
+                    </ul>
+                    <!-- <form class="d-flex">
+                  <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                  <button class="btn btn-outline-success" type="submit">Search</button>
+                </form> -->
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <h1>About me</h1>
+        <img src="assets/moi.jpeg" alt="picture of me">    
+        <p>Aspiring 3D artist turned web designer. I would describe myself as a pragmatic and efficient man. I enjoy playing various games, including card games, video games, board games, etc. Feel free to write me if you are in need of a comission!</p>
+        <a class="btn btn-dark" href="linkedIn" role="button">Resume on Linkedin</a>
```

Comment:
Missing link. Also, when using links to external pages, try to use `target='_blank'` property.